sdk/core/core-client/src/base64.browser.ts

```diff
@@ -40,3 +40,10 @@ export function decodeString(value: string): Uint8Array {
   }
   return arr;
 }
+
+/**
+ * Converts a uint8Array to a string.
+ */
+export function uint8ArrayToString(ab: Uint8Array): string {
+  return decodeURIComponent(escape(Array.from(ab, (item) => String.fromCharCode(item)).join("")));
```

Comment:
I'm still confused by the use of `escape` here, the closest hint I can see is from the [escape()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape) MDN page:

> Note: This function was used mostly for URL queries (the part of a URL following ?)—not for escaping ordinary String literals, which use the format "\xHH". (HH are two hexadecimal digits, and the form \xHH\xHH is used for higher-plane Unicode characters.)
>
>Escaped characters in String literals can be expanded by replacing the \x with %, then using the decodeURIComponent() function.


Is the idea that the claims contain *both* characters escaped with `\xHH` syntax and `%`? But it's still odd to me that we're escaping (e.g. "foo$" becomes "foo%24") just to then decode it back("foo%24" becomes "foo$" again)

Reply:
thank you! I’ll follow up asap

Reply:
yeah, `escape` wasn’t needed. I was having problems testing in the browser and this slipped through as I was writing this code, then I wasn’t remembering the reason I had `escape` on the first place. Things work well without this.